src/components/BaseSlider.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { ChangeEvent, FunctionComponent, ReactElement } from "react";
+import { Typography, Slider } from "@material-ui/core";
+
+export enum Sliders {
+  contrast,
+  brightness,
+}
+
+export const sliderConfig: {
+  [id: number]: SliderConfigItem;
+} = {
+  [Sliders.contrast]: {
+    name: "contrast",
+    initial: 100,
+    min: 0,
+    max: 200,
+  },
+
+  [Sliders.brightness]: {
+    name: "brightness",
+    initial: 100,
+    min: 0,
+    max: 200,
+  },
+};
+
+interface SliderConfigItem {
+  name: string;
+  initial: number;
+  min: number;
+  max: number;
+}
+
+interface Props {
+  value: number;
+  onChange: (arg0: string) => (arg1: ChangeEvent, arg2: number) => void;
+  slider: Sliders;
+}
+
+export const BaseSlider: FunctionComponent<Props> = ({
+  value,
+  slider,
+  onChange,
+}): ReactElement => {
+  const config = sliderConfig[slider];
+  return (
+    <>
+      <Typography
+        id="continuous-slider"
```

Comment:
I don't think the id should be set by the base component, ids should be unique and they won't be here - all sliders will share the same id... which means it's actually a class...